README.md

```diff
@@ -0,0 +1,17 @@
+The previous version of LLVM semantics is created by David and Chucky.
```

Comment:
Please use correct English.
Who are David and Chucky?  Again: keep your users in mind.